Formula/unxip.rb

```diff
@@ -0,0 +1,24 @@
+class Unxip < Formula
+  desc "Fast Xcode unarchiver"
+  homepage "https://github.com/saagarjha/unxip"
+  url "https://github.com/saagarjha/unxip.git", tag: "v1.1.3", revision: "b3ce1ab4728d4173390c97eddd28e821ad9e2974"
+  license "LGPL-3.0-only"
+  head "https://github.com/saagarjha/unxip.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on macos: :monterey
+  uses_from_macos "swift"
+
+  def install
+    system "swiftc", "-O", "-parse-as-library", "unxip.swift"
```

Comment:
why not use this kind of standardized build line

```suggestion
    system "swift", "build", "--disable-sandbox", "-c", "release"
```

Reply:
I got these build options via the maintainer's docs, but he'd also given me guidance to avoid using SPM here so that it wouldn't require an Xcode to build from source (that just the Command Line Tools will suffice).

Reply:
Rarely anyone using Homebrew builds from source since there are prebuilt binaries available

Reply:
Ok, I've switched to this standardized build line instead and added `depends_on xcode: :build` to the formula.